src/components/Wallet.tsx

```diff
@@ -1,12 +1,12 @@
 import React from 'react'
 import styled from 'styled-components'
-import { Box, GU, LoadingRing, textStyle, useTheme } from '@1hive/1hive-ui'
+import { GU, Box, LoadingRing, textStyle, useTheme } from '@1hive/1hive-ui'
```

Comment:
Just comment, order here dont mind the PR https://github.com/1Hive/gardens-ui/pull/747 will handle that automatic in the future